### DISCUSSION TITLE
Tagging tracks that are not organised by album folder

### DISCUSSION THREAD

**Discussion by oranjebloom** [complexlogic/rsgain#84] [Q&A] [answered]
I have stared using rsgain via the command line (Terminal) on Mac with the following setting:

`rsgain easy -p no_album /path/to/files/`

I am using this setting because I do not have my files segregated into separate sub-folders organised by album. Instead, the files are all lumped together in one large folder... and also because I read the instructions below that recommend this setting. 

_Easy Mode assumes that you have you have your music library organized by album, so that each album is contained in its own folder. The album gain calculations rely on this assumption. If you do not have your music library organized by album, you should disable the album tags because the calculated values will not be valid._

My two questions are as follows:

1.) Is this the correct setting for my case ... given that I want to ensure that all the tracks in the one large folder play at the same volume for the purpose of a streaming radio?
 
2.) When tagging new files that need to be added to this large folder, should these new files be added to the large directory first before running rsgain? ... Or can they be tagged separately and then added to the large folder containing the already tagged files?

**Comment by complexlogic** [comment]
> 1.) Is this the correct setting for my case ... given that I want to ensure that all the tracks in the one large folder play at the same volume for the purpose of a streaming radio?

Yes

> 2.) When tagging new files that need to be added to this large folder, should these new files be added to the large directory first before running rsgain? ... Or can they be tagged separately and then added to the large folder containing the already tagged files?

I recommend tagging new files separately in a different folder, then moving them into the large folder afterwards. If that's not practical in your workflow, you can add the new files directly to the large folder and use the `-S` option to skip tagging the files that have been previously tagged.

**Reply by oranjebloom** [reply]
Thank you for confirming this and for the advice regarding the `-S` option. I had read about that that but I thought that it wasn't possible to use the `-S` option when scanning a large folder unless I used a script. That's why I decided to look at using the rsgain plugin in MB Picard instead of via the Mac command line. 

To can a large folder of files in MB Picard with `-p no_album` and `-S`, I assume I need to attach a profile in the Replaygain 2.0 settings menu otherwise Picard will just tag files using `easy `album gain mode. I haven't been successful in finding instructions for how to apply different Replaygain tagging settings in Picard. Could you point me to any online resource for this?

I also assume that I can't use `-p no_album` and -`S` at the same time because one is for `easy` mode and the other is a` custom `tagging mode. Is this right?

**Reply by complexlogic** [reply, marked as answer]
> To can a large folder of files in MB Picard with -p no_album and -S, I assume I need to attach a profile in the Replaygain 2.0 settings menu otherwise Picard will just tag files using easy album gain mode. I haven't been successful in finding instructions for how to apply different Replaygain tagging settings in Picard. Could you point me to any online resource for this?

The Picard plugin uses rsgain Custom Mode underneath, so you can't use a scan preset. However, most of the settings from the presets are exposed in the plugin settings. Go to Options->Options->Plugins->ReplayGain 2.0. In that window you can change the settings. For example, disable the album gain.

> I also assume that I can't use -p no_album and -S at the same time because one is for easy mode and the other is acustomtagging mode. Is this right?

`-S` can be used in both Easy Mode and Custom Mode. `-p no_album` is Easy Mode only.

**Reply by oranjebloom** [reply]
Very clear. Thanks so much!
